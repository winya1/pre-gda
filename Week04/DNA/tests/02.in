6       A  AG AT TG AATGTATC CA
8
SmallA  2  0  0  0  0        0
MediumA 5  0  1  0  0        0
BigA    10 0  0  0  0        0
Short   5  4  5  6  0        0
Joe     2  1  1  1  5        1
Alice   7  0  1  1  1        5
Ater    1  0  3  0  0        0
AterTwo 2  0  3  0  0        0
10
A
AA
AAAAAAAAAA
AAAAAAAAAAAA
ATATAT
ATATATAA
AAAATAAAAA
AGAGAGAGAAAAATATATATATGTGTGTGTGTG
AATGTATCAATGTATCAATGTATCAATGTATCAATGTATCAG
AAAAAAATGTATCACACACACA
