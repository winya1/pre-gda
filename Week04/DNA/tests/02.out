Case #1: No match
Case #2: SmallA
Case #3: BigA
Case #4: No match
Case #5: Ater
Case #6: AterTwo
Case #7: MediumA
Case #8: Short
Case #9: Joe
Case #10: Alice
